org.metaborg.meta.sdf3.ext.statix/metaborg.yaml

```diff
@@ -0,0 +1,47 @@
+---
+id: org.metaborg:org.metaborg.meta.sdf3.ext.statix:${metaborgVersion}
+name: Sdf3ExtStatix
+metaborgVersion: 2.6.0-SNAPSHOT
+
+dependencies:
+  compile:
+  - org.metaborg:org.metaborg.meta.lang.esv:${metaborgVersion}
+  source:
+  - org.metaborg:meta.lib.spoofax:${metaborgVersion}
+  - org.metaborg:org.metaborg.meta.lang.template:${metaborgVersion}
+  - org.metaborg:statix.lang:${metaborgVersion}
+  - org.metaborg:statix.runtime:${metaborgVersion}
+pardonedLanguages:
```

Comment:
This file is missing a `contributions` section to integrate with the language. See for example https://github.com/metaborg/nabl/blob/master/nabl2.ext.dynsem/metaborg.yaml#L15-L17.